### ISSUE TITLE
randomInt window does not exist on nodejs

### ISSUE THREAD

**Issue by PatrickGeyer** [subhamg/unique-username-generator#11]
missing polypill for window object

**Comment by dylangolow** [comment]
You can import polyfills to fix this when using the package in Node.js.

Window polyfill: https://www.npmjs.com/package/node-window-polyfill
Crypto: https://www.npmjs.com/package/window-crypto

You can install them
```bash yarn add node-window-polyfill/register window-crypto```

 import wherever you want to user this package by adding these import lines:
```typescript
...
import "node-window-polyfill/register";
import "window-crypto";
...
```

**Comment by subhamg** [comment]
Hi @PatrickGeyer,

Good news! The issue you reported has been resolved in version `1.2.0` of our library. We have updated the code to ensure compatibility with Node.js, and the randomInt function now works seamlessly in server-side environments.

Feel free to install the latest version (1.2.0), and you should no longer encounter the "randomInt window does not exist on Node.js" error.

Thank you for bringing this to our attention, and we appreciate your support!
